packages/components/src/text-control-with-affixes/index.js

```diff
@@ -0,0 +1,54 @@
+/** @format */
```

Comment:
This file is using only spaces, not tabs – our JS files should be using tab indentation.

Reply:
Fixed in cc92f19.